allure-generator/src/main/javascript/plugins/tab-graph/GraphsView.js

```diff
@@ -5,26 +5,26 @@ import {className} from '../../decorators';
 import DurationChart from './charts/DurationChart';
 import StatusChart from './charts/StatusChart';
 import SeverityChart from './charts/SeverityChart';
-
+import t from '../../helpers/t';
```

Comment:
It could be 

```
import translation from '../../helpers/t';
```

In handlebars `t` looks nice, but in JS code it is better to use full name. How do you think?